reference/core/ingress-controller.md

```diff
@@ -184,7 +183,7 @@ spec:
   service: exampleservice:8080
 ```
 
-### [Name based virtual hosting](https://kubernetes.io/docs/concepts/services-networking/ingress/#name-based-virtual-hosting) with an Ambassador Edge Stack ID
+### Name based virtual hosting with an Ambassador Edge Stack ID
```

Comment:
Title case?

Reply:
addressed in another PR... this is for priority issues and not small typos and such